golem-cluster.yaml

```diff
@@ -25,7 +25,7 @@ provider:
     #payment_network: "polygon"
 
     # Maximum amount of GLMs that's going to be spent for the whole cluster
-    total_budget: 1
+    total_budget: 5
```

Comment:
What about `mini` and `full` configs?

Reply:
in `mini` there is no total_budget (default is used), `full` already has `5` :)